lib/ProductOpener/Ingredients.pm

```diff
@@ -741,23 +741,23 @@ fr => [
 ],
 
 es => [
-'ingredientes'(\s*)(\s|-|:|\r|\n)+
+'ingredientes'(\s*)(\s|-|:|\r|\n)+,
```

Comment:
This is a string, the delimiters ' ... ' need to be at the beginning and end :

'ingredientes'(\s*)(\s|-|:|\r|\n)+, --> 'ingredientes(\s*)(\s|-|:|\r|\n)+',

--> same for all changes below

Otherwise it won't compile.

Reply:
fixed